azure-servicefabric/azure/servicefabric/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "6.0"
+VERSION = "6"
```

Comment:
Is this okay? I'm not sure if we treat `6.0` differently from `6`

Reply:
No, I will change to 6.0.1.